### ISSUE TITLE
FileLoadException in BundlerMinfier.Core 3.2.435

### ISSUE THREAD

**Issue by martincostello** [madskristensen/BundlerMinifier#450]
I've just tried to update a ASP.NET Core 3.0 project to use the new version for `netcoreapp3.0` from #443 and got this error:

```
Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'System.Linq, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)                                                                              at BundlerMinifier.BundleHandler.TryGetBundles(String configFile, IEnumerable`1& bundles)                              at BundlerMinifier.Program.GetConfigFileFromArgs(String[] args, String& configPath)                                    at BundlerMinifier.Program.Main(String[] args)
```

I guess there's something missing somewhere or misconfigured in the package dependencies that meant that the tests didn't pick it up.

/cc @rockstardev

**Comment by martincostello** [comment]
PR that demonstrates the issue: https://github.com/martincostello/aspnet-core-pseudo-localization/pull/38

**Comment by rockstardev** [comment]
In that repository you don't seem to be referencing 3.2.435... if you look at csproj:

https://github.com/martincostello/aspnet-core-pseudo-localization/blob/master/src/TodoApp/TodoApp.csproj#L18

This seems to be some documented problem with how projects define bindings for certain DLLs:

https://github.com/dotnet/corefx/issues/32757

https://stackoverflow.com/questions/42755274/visual-studio-2017-could-not-load-file-or-assembly-system-runtime-version-4

**Comment by martincostello** [comment]
That’s the master branch, the change is in the linked PR’s branch.

**Comment by martincostello** [comment]
#453 demonstrates the same issue as in the macOS build for the linked PR for this: https://martincostello.visualstudio.com/aspnet-core-pseudo-localization/_build/results?buildId=2663&view=logs&jobId=a5e52b91-c83f-5429-4a68-c246fc63a4f7&taskId=0eb4d043-45d3-5a4a-c7f2-f7c2d6566b01&lineStart=50&lineEnd=51&colStart=1&colEnd=1

**Comment by abazanov** [comment]
@martincostello have you ever been able to fix this issue? I have a project where we are using `BundlerMinifier.Core 3.2.435` and I am getting `Could not load file or assembly 'System.Linq, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'`

Many thanks.

**Comment by martincostello** [comment]
I’m afraid not. I’m not sure why it’s broken.

**Comment by abazanov** [comment]
@martincostello I cannot find System.Linq 4.2.0.0 on the NuGet Gallery https://www.nuget.org/packages/System.Linq/ so it is trying to download a version of System.Linq that does not exist it seems.

**Comment by abazanov** [comment]
@martincostello we fixed it. I had to go through all my SDKs and uninstall some old ones, because there one Runtime version that was loading a bad System.Linq.dll file. So delete some of your old versions of the SDKs especially the one with a `6` in its build version.

**Comment by martincostello** [comment]
Not really feasible on Azure DevOps Pipelines (and other) hosted CI solutions 😞

**Comment by abazanov** [comment]
The version that I had to get rid of on my work machine was SDK 2.0.6 which had 2.1.104 runtime. Fixed now. Azure DevOps will have the latest stuff installed, won't it?

**Comment by MrAPierce** [comment]
I'm getting the same error about Linq assembly not found in Azure DevOps. Any work around for this issue?

**Comment by abazanov** [comment]
@MrAPierce yes, there is. You need to go into your Control Panel (Windows) don't know what it would be for Mac, and start deleting some of the old versions of the .NET Core SDK (see image below for example). The image shows the versions of the SDK that I have on my PC and it works fine. 

![image](https://user-images.githubusercontent.com/5940942/69638066-e0e94200-1051-11ea-8f60-ffad263cd92c.png)

**Comment by martincostello** [comment]
That won’t work in AzDO though.

**Comment by drmathias** [comment]
Stopped getting the exception after uninstalling .NET Core SDK 2.1.202, thanks @abazanov

**Comment by ThomasArdal** [comment]
I uninstalled all 1.x versions of .NET Core to stop getting this error.

**Comment by michaeldevenney** [comment]
Anyone know what the fix is for Azure DevOps deployments?

**Comment by eriksendc** [comment]
We're just finishing up going to ASP .NET Core 3.1 from 2.2. Included in this work csproj now references BundlerMinifier.Core version 3.2.435.

Initially my full build was failing with this error: https://github.com/madskristensen/BundlerMinifier/issues/453

At first, since the error messages included "The framework 'Microsoft.NETCore.App', version '2.0.0' was not found." I tried inserting a Use .NET Core task and set the Version to 2.0.0. When I tried running the build again I got the exact error mentioned first in this issue by @martincostello 

I then switched the Version to 2.2.106, since that's what we were using to build prior to upgrading to 3.1.

To be clear, I now have two Use .NET Core tasks. The first one gets the 3.1.201 SDK. Then I have a Restore and a Build task, then I have the second Use .NET Core task that gets 2.2.106, This seems to make dotnet bundle happy.

I know this is a funky workaround, but at least I'm building in Azure Dev Ops Pipelines like I was with 2.2.
